logs/ingress/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "ingress"
+version = "0.1.0"
+authors = ["Joseph Azevedo <joseph.az@gatech.edu>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+lazy_static = "1.4.0"
+log = "0.4.8"
+env_logger = "0.7.1"
+tokio = { version = "0.2.20", features = ["macros"] }
+futures = "0.3.4"
+logs_lib = { path = "../lib" }
+serde = { version = "1.0.111", features = ["derive"] }
+serde_json = "1.0.55"
+backoff = "0.1.6"
+reqwest = "0.10.6"
```

Comment:
Unless it's necessary to have these exact versions, you can just do X.Y and if the library writers do semantic versioning properly, you'll get the newest version that doesn't break the API.